components/layout/header/LayoutHeaderAccountInner.vue

```diff
@@ -0,0 +1,99 @@
+<script setup lang="ts">
+withDefaults(
+    defineProps<{
+      signedIn?: boolean;
+    }>(),
+    {
+        signedIn: false,
+    },
+);
+</script>
+
+<template>
+    <UiDropdownMenu>
+        <slot name="icon-trigger">
```

Comment:
```suggestion
        <slot name="trigger">
```

I think a more generic name could be better if for example a project doesn't want to have an icon as trigger